### ISSUE TITLE
Not loading files correctly, seems to be creating cassettes for each folder in pck

### ISSUE THREAD

**Issue by Hazelberry** [Moose1002/Cozy-Cassettes#1]
As the title says it seems to be loading the folders from the pck as cassettes instead of loading the folders inside my Cassettes folder.

I tested this both through a manual installation of GDWeave + Lure + Cozy-Cassettes, and installing all of those through R2ModManager (with the manual installation removed). I made sure that they were the only mods installed and was still unable to get the mod to work correctly.

Here are screenshots of what it looks like in game, along with one of my cassette folders (in the R2ModManager installation method, but the file path for the manual installation method matched the example screenshot given for the mod). All music files used are MP3 format typically ranging in size from 3MB to 7MB.

![webfishing_3udFj0y3A7](https://github.com/user-attachments/assets/de7190ea-58b2-43ec-9f32-8b0288707a72)
![webfishing_GG3YnqXFdA](https://github.com/user-attachments/assets/eb29a899-922c-4b42-b1ec-3ccc301ed68c)
![explorer_SYtXlg2RpS](https://github.com/user-attachments/assets/3754620e-e994-4813-9feb-a7320406535c)

**Comment by Moose1002** [comment]
Something seems to have renamed the mod directory. What modloader are you using?

Cozy Cassettes tries to find your Cassettes folder at "/GDWeave/mods/Moose1002.CozyCassettes/Cassettes/"
Somehow yours got renamed to "/GDWeave/mods/Moose1002-Cozy_Cassettes/Cassettes/"

A simple rename will fix your issue, but I'll look into what's causing it. Thanks for the report!

**Comment by Hazelberry** [comment]
Ah that's it! Thanks so much for the quick reply.

I downloaded it through R2ModMan from the Thunderstore, I guess it didn't like the period and replaced it with a hyphen.

Went and downloaded your mod manually from the Thunderstore page instead and doing it that way still preserved the period. Probably would be worth it to either remove the period or add a check for the hyphenated version.

EDIT: I think I figured out part of the issue. R2ModManager stores a cached version of mods using the zip name, which seems to be hyphenated by Thunderstore. It does correctly have the folder with a period in the name within that cache, but when creating a mod profile it uses the hyphenated zip name instead of the proper folder period name. This seems to be a built in function of R2ModManager and/or Thunderstore, so I definitely think including a check for this alternate folder name would be the best idea for compatibility.

I tried changing the cached folder's name to use the period instead of a hyphen but that seems to break R2ModManager's ability to read the folder.

![explorer_Uk7o6FbzD7](https://github.com/user-attachments/assets/8556fe7b-da7b-47a3-abdf-072c051aadc6)
![explorer_x1XS1TthbI](https://github.com/user-attachments/assets/189ffb6c-cf29-4689-87a5-e427bc787e22)
![explorer_IlY12CtiEx](https://github.com/user-attachments/assets/f0b112b0-6398-452e-b299-c2eaeb146f71)
